lib/Runtime/Language/JavascriptOperators.cpp

```diff
@@ -4818,6 +4818,11 @@ namespace Js
 #endif
         }
 
+        if (result == FALSE && (propertyOperationFlags & PropertyOperation_StrictMode))
+        {
+            JavascriptError::ThrowCantDelete(propertyOperationFlags, scriptContext, TaggedInt::ToString(index, scriptContext)->GetString());
+            return scriptContext->GetLibrary()->CreateBoolean(FALSE);
```

Comment:
Also, no real point to have this return statement duplicated, we can fall through to the below one and get the same behavior.